tests/test_ensembles/test_direct_ensemble.py

```diff
@@ -28,6 +28,21 @@
 from tests.test_pipeline.utils import assert_pipeline_predicts
 
 
+def check_backtest_return_type(backtest_result: dict):
```

Comment:
This function copied several times, we can generalize it a bit more and place into common.